CloneResponsive/styles.css

```diff
@@ -0,0 +1,812 @@
+* {
+  font-family: Inter !important;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  background-color: #f2f4f7;
+}
+.logo {
```

Comment:
You can see the logo block is not pixel-perfect  with sidebar.
And you can see that elements of sidebar are partially out of sidebar borders. It can create problems in future
![Screenshot 2023-04-10 at 21 41 12](https://user-images.githubusercontent.com/70685561/230959528-55ddd72f-78da-4d7c-bd25-9853514487da.png)